### PR TITLE
#195 Remove cip marks after structure changing + revert autorecalc cip

### PR DESCRIPTION
**Pull request by irinatelnova**

### CODE REVIEW

**Review by AndreiMazol — APPROVED** [review]

File: packages/ketcher-react/src/script/editor/operations/sgroup/index.ts [hunk 1]
```diff
@@ -59,10 +59,12 @@ class SGroupCreate extends BaseOperation {
 
 class SGroupDelete extends BaseOperation {
   data: Data
+  checkEmptiness: boolean
 
-  constructor(sgroupId?: any) {
+  constructor(sgroupId?: any, checkEmptiness = true) {
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/editor/operations/sgroup/index.ts:64]: please rename to 'validate' instead

**Review by AndreiMazol — CHANGES_REQUESTED** [review]

File: packages/ketcher-react/src/script/ui/state/server/index.js [hunk 2]
```diff
@@ -147,10 +147,14 @@ export function serverTransform(method, data, struct) {
...
       .then(res =>
         dispatch(
-          load(res.struct, {
-            rescale: method === 'layout',
-            reactionRelayout: method === 'clean'
-          })
+          load(
+            res.struct,
+            {
+              rescale: method === 'layout',
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/ui/state/server/index.js:153]: look strange. the solution  should be discussed

File: packages/ketcher-react/src/script/editor/operations/sgroup/index.ts (hunk 1 above)
Comment (StartsevDS) [reply packages/ketcher-react/src/script/editor/operations/sgroup/index.ts:64]: Naming booleans starting with "is" considered a good practice. In this case "isValid" or "isEmpty" would be self-descriptive.

**Review by AndreiMazol — COMMENTED** [review]
please fix comments

**Review by MagomedGasanov — APPROVED** [review]

**Pull request closed without merging**